utils/llm_client.py

```
@@ -0,0 +1,169 @@
+                if response.status != 200:
+                    error_text = await response.text()
+                    raise RuntimeError(f"Anthropic API error: {response.status} - {error_text}")
+                
+                result = await response.json()
+                
+                if "content" not in result or not result["content"]:
+                    raise RuntimeError("Invalid Anthropic response format")
+                
+                return result["content"][0]["text"]
+
+def create_llm_client(model: str) -> LLMClient:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)